#!/usr/bin/python3

def raise_exception():
    """_summary_
    """
    raise TypeError("error")
